src/pages/Jihong/MainComponents/Section/Section.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import FILTER_LIST from '../../constantData';
+
+const Section = ({ isClick, totalNumberItems }) => {
+  return (
+    <section>
+      <div className="sectionLeft">총 {totalNumberItems} 개</div>
+      <div className="sectionRight">
+        {FILTER_LIST.map((filterList, idx) => {
+          return (
+            <button
+              type="button"
+              key={idx}
+              name={filterList.filterListName}
+              onClick={e => isClick(e)}
```

Comment:
```suggestion
              onClick={isClick}
```
event를 인자로 전달할 때는 isClick만 넣어도 위 코드와 동일하게 적용됩니다!